activerecord/lib/active_record/core.rb

```diff
@@ -85,6 +85,15 @@ def self.configurations
       mattr_accessor :dump_schema_after_migration, instance_writer: false
       self.dump_schema_after_migration = true
 
+      ##
+      # :singleton-method:
+      # Specifies which database schemas to dump when calling db:structure:dump.
+      # If :schema_search_path, it will dumps any schemas listed in schema_search_path.
+      # Use :all to always dumps all schemas regardless of the schema_search_path.
+      # A string of comma separated schemas can also be used to pass a custom list of schemas.
```

Comment:
I was trying to model it after the documentation for schema_format (above) which doesn't specify the default in the doc. Should I change both?